### DISCUSSION TITLE
Question on the "dotted" keyword in Lines() object

### DISCUSSION THREAD

**Discussion by jkissi** [marcomusy/vedo#692] [Q&A] [answered]
Hi Marco!!
It's been a while, I hope you are well! 
I am trying to get the `Lines()` object to show a dotted line using the keyword but the lines always seem to come out _solid_.  Could you please show a quick example of this working? In the docs, it states that its a simple boolean...this is correct, no? 

Cheers 
j

**Comment by marcomusy** [comment, marked as answer]
Hi, the dotted feature is less than perfect... You may explore one of these options:
```python
from vedo import *
pts = [[1, 0, 0], [5, 2, 0], [3, 3, 1]]
ln = Line(pts, c='r', lw=5).pattern('- -', repeats=10)
ln.show(axes=1).close()

ln = DashedLine(pts, c='r', lw=4)
ln.show(axes=1).close()

ln = Line([5, 2, 0], [3, 3, 1], res=50).ps(5).c('r').lighting("off")
ln.show(axes=1).close()
```

**Reply by jkissi** [reply]
Just to be sure, I can't use the `.pattern()` example with the `Lines()` object, can I?
